.github/actions/enforce-pr-conventions/action.yml

```diff
@@ -0,0 +1,77 @@
+name: enforce-pr-conventions
+
+description: Check if the branch name and pull request title follow conventional checks.
+
+inputs:
+  jira-project-name:
+    description: 'The JIRA project name.'
+    required: false
+  valid-branch-regex:
```

Comment:
Error messages used "regex" and inputs use "regexp": maybe this should be consistent

Reply:
Done